packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -270,13 +268,11 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
         icon: getCssVarColor('palette-neutral-400'),
       },
       background: {
-        body: getCssVarColor('palette-common-black'),
-        surface: getCssVarColor('palette-neutral-900'),
-        popup: getCssVarColor('palette-common-black'),
-        level1: getCssVarColor('palette-neutral-800'),
-        level2: getCssVarColor('palette-neutral-700'),
-        level3: getCssVarColor('palette-neutral-600'),
-        tooltip: getCssVarColor('palette-neutral-600'),
+        level0: getCssVarColor('palette-common-black'), // Former body and also intended to be used for popup
+        level1: getCssVarColor('palette-neutral-900'), // Former surface
+        level2: getCssVarColor('palette-neutral-800'), // Former level1
+        level3: getCssVarColor('palette-neutral-700'), // Former level2
+        level4: getCssVarColor('palette-neutral-600'), // Former level3 and also intended to be used for tooltip
         backdrop: `rgba(${getCssVar(
```

Comment:
@zanivan What's about `backdrop`?

Reply:
IMO I think we should keep it as `backdrop`.

If we insert it in this scale, it'd be like a `level0.5`, what is not so good. Moreover, though `backdrop` is there as a background, I believe it works like a filter rather than an actual background color.

Reply:
Or we rename it to `translucent`? The reason is `backdrop` does not seem to be descriptive as we want and I afraid that this will create inconsistency for us.

The `translucent` can be extended to `translucent1`, `translucent2`, etc.

Reply:
It makes sense, and I like the possibility of extending these! So we can change it to `translucent` 👍